LICENSE

```diff
@@ -1,29 +0,0 @@
-BSD 3-Clause License
-
-Copyright (c) 2017, SeatGeek
-All rights reserved.
-
-Redistribution and use in source and binary forms, with or without
-modification, are permitted provided that the following conditions are met:
-
-* Redistributions of source code must retain the above copyright notice, this
-  list of conditions and the following disclaimer.
```

Comment:
Also, deleting this file is against the license.

Reply:
True, thank you for pointing out.
I've deleted forked repo since it become a mess.
I'll set up properly next time when its ready for production.